components/scream/src/physics/shoc/tests/shoc_update_prognostics_implicit_tests.cpp

```diff
@@ -372,19 +381,14 @@ struct UnitWrap::UnitTest<D>::TestUpdatePrognosticsImplicit {
       UpdatePrognosticsImplicitData& d_f90 = f90_data[i];
       UpdatePrognosticsImplicitData& d_cxx = cxx_data[i];
       for (Int k = 0; k < d_f90.total(d_f90.thetal); ++k) {
-        REQUIRE(d_f90.total(d_f90.thetal) == d_cxx.total(d_cxx.thetal));
         REQUIRE(d_f90.thetal[k] == d_cxx.thetal[k]);
```

Comment:
Why did you delete these `total` checks?

Reply:
Ah, I meant to pull them out of the for-loop (since they do not depend on k) but forgot to paste above.. They're back now.